app/api/schema/attendees.py

```diff
@@ -51,6 +51,7 @@ def validate_json(self, data, original_data):
     facebook = fields.Url(allow_none=True)
     github = fields.Url(allow_none=True)
     gender = fields.Str(allow_none=True)
+    accept_video_recording = fields.Boolean(default=True)
```

Comment:
allow_none=True, we cannot assume previous attendees accepted this requirment, null is the better option for them